Tests/Test Helpers/Editor/EditorVRTestInitializer.cs

```diff
@@ -0,0 +1,35 @@
+﻿using NUnit.Framework;
+using UnityEditor.Experimental.EditorVR.Core;
+
+namespace UnityEditor.Experimental.EditorVR.Tests.Core
+{
+	[SetUpFixture]
+	public class EditorVRTestInitializer
+	{
+		EditingContextManagerSettings projectSettingsBackup;
+		EditingContextManagerSettings userSettingsBackup;
+
+		[OneTimeSetUp]
+		public void SetupBeforeAllTests()
+		{
+			projectSettingsBackup = EditingContextManager.LoadProjectSettings();
+			userSettingsBackup = EditingContextManager.LoadUserSettings();
+
+			EditingContextManager.ShowEditorVR();
+		}
+
+		[OneTimeTearDown]
+		public void CleanupAfterAllTests()
+		{
+			EditingContextManager.SaveProjectSettings(projectSettingsBackup);
+			EditingContextManager.SaveUserSettings(userSettingsBackup);
```

Comment:
Is this necessary yet? I'm wondering if it's possible to fake the settings so that we don't actually touch user settings. Not sure if that would be more trouble than it's worth.

Reply:
sounds like a lot of trouble imho